elf/input-files.cc

```diff
@@ -120,9 +120,13 @@ ObjectFile<E>::read_note_gnu_property(Context<E> &ctx, const ElfShdr<E> &shdr) {
       u32 size = *(U32<E> *)(desc.data() + 4);
       desc = desc.substr(8);
 
-      // All currently defined .note.gnu.property use 32-bit values.
+      // The majority of currently defined .note.gnu.property use 32-bit values.
```

Comment:
nit: Rui usually keeps stuff formatted for 80 columns so might want to update the line wrapping.